lib/slack.ml

```diff
@@ -152,7 +172,7 @@ let generate_issue_comment_notification notification channel =
     sprintf "<%s|[%s]> *%s* <%s|%s> on #%d %s" repository.url repository.full_name sender.login comment.html_url
       action_str number (pp_link ~url:issue.html_url title)
   in
-  make_message ~text:summary ~attachments:(markdown_text_attachment ~footer:None comment.body) ~channel ()
+  make_message ~text:summary ?attachments:(format_attachments slack_match_func None @@ Some comment.body) ~channel ()
```

Comment:
the function application is confusing, with the removal of named arg and addition of @@